src/odemis/driver/test/semnidaq_test.py

```diff
@@ -1036,16 +1075,22 @@ def test_sync_ai_ci(self):
 
         # set softwareTrigger on the first detector to be subscribed
         self.sed.data.synchronizedOn(self.sed.softwareTrigger)
+        # self.counter.data.synchronizedOn(self.counter.softwareTrigger)
```

Comment:
There are a few commented out lines in here. Do you want to keep them as reference? We could use git history for that right?